### ISSUE TITLE
[Site Request] Comic Park

### ISSUE THREAD

**Issue by zeoint** [manga-download/hakuneko#6868]
**Name of the website**
Comic Park

**Website urls (examples below)**
- Site: https://comicpark.org/
- Manga List: https://comicpark.org/search
- Manga example: https://comicpark.org/title/224189-en-i-became-the-wife-of-the-male-lead
- Chapter online viewer example: https://comicpark.org/title/224189-en-i-became-the-wife-of-the-male-lead/6344711-chapter-1

**Languages**
English

**Website relationship*

- https://mangapark.com
- https://mangapark.net
- https://mangapark.org
- https://mangapark.me
- https://mangapark.io
- https://mangapark.to
- https://comicpark.org
- https://comicpark.to
- https://readpark.org
- https://readpark.net
- https://parkmanga.com
- https://parkmanga.net
- https://parkmanga.org
- https://mpark.to

**Additional details**

**Comment by MikeZeDev** [comment]
Do we need another mangapark clone/mirror?

**Comment by zeoint** [comment]
Okay, I didn't know there was already mangapark in the connectors list, I pasted the above comicpark link using clipboard, that did not work and searched for comic park in the connectors list and I couldn't find it, so I opened a site request here. But nevermind it.

Maybe allow support for clipboard pasting of any links in hakuneko, as only the mirror name changes, and also when searched for above names in connectors list, show manga park.
